### PR TITLE
adding ui test for checking sanctions

### PR DESCRIPTION
**Pull request by sekharpanja**
LTD-2110 UI test to check sanctions

### CODE REVIEW

File: ui_tests/caseworker/features/sanctions.feature [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@all @internal @sactions
+Feature: I want to check sactions matches
+  As a logged in government user
+  I want to check that the sanction match is highlighted
+  For a case is created with a name on it that has a sanction
+
+  @check_sanctions
```
Comment (saruniitr) [inline ui_tests/caseworker/features/sanctions.feature:7]: this need not be present

File: ui_tests/caseworker/features/sanctions.feature [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  Scenario: view a case note added by caseworker
```
Comment (saruniitr) [inline ui_tests/caseworker/features/sanctions.feature:8]: please update this to describe the scenario

File: ui_tests/caseworker/step_defs/test_sanctions.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+@given("For a case is created with a name on it that has a sanction")  # noqa
+def sactioned_case(driver, sso_sign_in):  # noqa
+    pass
+
+
+@then(parsers.parse('I should see that the sanction match is highlighted as "{end_user_name}"'))
+def check_saction_highlighted(driver, end_user_name):  # noqa
+    sanctions_table = driver.find_element(by=By.ID, value="table-sanction-matches")
+    name_element = sanctions_table.find_element(by=By.XPATH, value="//tbody/tr/td[4]")
```
Comment (saruniitr) [inline ui_tests/caseworker/step_defs/test_sanctions.py:21]: don't remember exactly, are we only matching on name or name and address?
If address is also included then please add assertions for address as well.

Comment (sekharpanja) [reply ui_tests/caseworker/step_defs/test_sanctions.py:21]: There is no specific requirement to check address.

Comment (saruniitr) [reply ui_tests/caseworker/step_defs/test_sanctions.py:21]: but do we use address also to highlight an entry as sanctions match or just name?

**Review by saruniitr — APPROVED** [review]

**Pull request merged**